chat-client-ui-types/src/uiContracts.ts

```diff
@@ -67,6 +80,24 @@ export interface AuthFollowUpClickedMessage {
     params: AuthFollowUpClickedParams
 }
 
+export interface CopyCodeToClipboardParams {
```

Comment:
Why do we need those in UI contracts? Will extension need to know the telemetry event to publish it to DWH?

Reply:
It's not necessary, but I think it's better to have a format that the client conforms to and possible future requirements where the data are expected

Reply:
It's more for the chat-client to conform it's data before sending to Toolkit, updated to reflect this comment in the adjacent PR https://github.com/aws/language-servers/pull/249#discussion_r1603316608